### PR TITLE
Fix the dowload api bug of issue #748

### PR DESCRIPTION
**Pull request by wudeyong**
Fix the bug of download API by using the driveItem unique identifier to locate a file**#748**

### CODE REVIEW

**Comment by rgregg** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/rest-api/api/driveitem_get_content.md](https://github.com/wudeyong/onedrive-api-docs/blob/patch-1/docs/rest-api/api/driveitem_get_content.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/onedrive/developer/rest-api/api/driveitem_get_content?branch=pr-en-us-749) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C12%5C5%5C1b9326b4-eb89-99a2-12be-09cf82b65094%5CPullRequest%5C201712050818124948-749%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=j%2BLPl1UrbvFUPlnJAGORFxFR1SpJq0YyjmzGTodLGLM%3D&st=2017-12-05T08%3A14%3A43Z&se=2018-01-05T08%3A19%3A43Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/rest-api/api/driveitem_get_content.md [hunk 1]
```diff
@@ -23,7 +23,7 @@ One of the following permissions is required to call this API. To learn more, in
 <!-- { "blockType": "ignored" } -->
 
 ```http
-GET /drives/items/{item-id}/content
+GET /drives/{driver-id}/items/{item-id}/content
```
Comment (ificator) [inline docs/rest-api/api/driveitem_get_content.md]: To be more consistent with the other examples, can we swap to `GET /drive/items/{item-id}/content` instead?

Comment (wudeyong) [reply docs/rest-api/api/driveitem_get_content.md]: I have try to use  GET /drives/items/{item-id}/content ,it return a error code of 400,and the message is : 
```
 {
  "error": {
    "code": "BadRequest",
    "message": "Resource not found for the segment '01FASBQH4B3TCO37AHZRCLRP3CVRSN34QX'.",
    "innerError": {
      "request-id": "4c90ff6a-37df-40cd-8436-e8e93fd087de",
      "date": "2017-12-07T09:07:46"
    }
  }
}
```

But I try to use **  GET /drives/{driver-id}/items/{item-id}/content **,it return a correct response.
So maybe the right api is  ** GET /drives/{driver-id}/items/{item-id}/content **

thanks

Comment (rgregg) [reply docs/rest-api/api/driveitem_get_content.md]: I've updated this to be more accurate. The /drive/items example is covered already through /me/drive/items in the list of examples, so the right fix here is what @wudeyong proposed. My only change was to rename "drives-id" to "drive-id" to match the way we reference that in other places in the content. I'll accept after the test run completes.

**Comment by rgregg** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/rest-api/api/driveitem_get_content.md](https://github.com/wudeyong/onedrive-api-docs/blob/patch-1/docs/rest-api/api/driveitem_get_content.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/onedrive/developer/rest-api/api/driveitem_get_content?branch=pr-en-us-749) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C12%5C7%5C1b9326b4-eb89-99a2-12be-09cf82b65094%5CPullRequest%5C201712072045146167-749%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=XObDHKjgJijgde3Eod3ZN54IOcFo6MxV9OFkFFMLk2Q%3D&st=2017-12-07T20%3A41%3A42Z&se=2018-01-07T20%3A46%3A42Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**